frontend_vendors/src/index.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
   // INFO - CH - Keep the line below or <Link> components will fail
   // See https://github.com/tracim/tracim/issues/3999
   'react-router-dom': require('react-router-dom'),
+  'reactstrap': require('reactstrap'),
```

Comment:
if added to frontend_vendor, you can remove it from the folders that import it

Reply:
What do you mean? Should we remove reactstrap from the list of dependencies of the apps that use it?
It seems strangle to do this while they also import reactstrap, no?